rocAL/rocAL/include/api/rocal_api_types.h

```diff
@@ -128,7 +128,8 @@ enum RocalTensorOutputType
 {
     ROCAL_FP32 = 0,
     ROCAL_FP16 = 1,
-    ROCAL_U8   = 2,
+    ROCAL_UINT8 = 2,
+    ROCAL_INT8 = 3
```

Comment:
We can either expand all datatypes or have short forms for each of them
like ROCAL_FLOAT32, ROCAL_FLOAT16, ROCAL_UINT8, ROCAL_INT8
or
ROCAL_FP32, ROCAL_FP16, ROCAL_U8, ROCAL_I8

Please check if we can make this change

Reply:
I feel this is not required

Reply:
@shobana-mcw can you please share your thoughts on this?